SWLOR.Game.Server/Feature/ShipModuleDefinition/CombatLaserModuleDefinition.cs

```diff
@@ -21,18 +21,30 @@ public class CombatLaserModuleDefinition: IShipModuleListDefinition
             CombatLaser("com_laser_2", "Combat Laser II", "Cmbt Laser II", "Deals 17 thermal DMG to your target.", 3, 5f, 12, 17);
             CombatLaser("com_laser_3", "Combat Laser III", "Cmbt Laser III", "Deals 21 thermal DMG to your target.", 4, 6f, 15, 21);
             CombatLaser("com_laser_4", "Combat Laser IV", "Cmbt Laser IV", "Deals 26 thermal DMG to your target.", 5, 7f, 18, 26);
+            CombatLaser("cap_pdl_1", "Capital Point Defense I", "Cap PDL I", "A corvette's fast firing point defense. 15 DMG.", 5, 2f, 2, 15);
```

Comment:
I think these are NPC items but let's go ahead and keep these focused on functionality and exclude any flavor text. Can you change the descriptions to be in line with the existing laser descriptions? E.G:

"Deals 15 thermal DMG to your target."